bepasty/utils/permissions.py

```diff
@@ -54,7 +54,7 @@ def get_permissions():
 def get_permission_icons():
     return [
         (permission, permission_icons[permission])
-        for permission in get_permissions()
+        for permission in get_permissions() if permission
```

Comment:
if you set no permissions, get_permissions will return [''] and because permission_icons has no key '', it will raise an IndexError